Auth0/_ObjectiveAuthenticationAPI.swift

```diff
@@ -47,6 +47,13 @@ public class _ObjectiveAuthenticationAPI: NSObject {
             .login(usernameOrEmail: username, password: password, connection: connection, scope: scope, parameters: parameters ?? [:])
             .start(handleResult(callback: callback))
     }
+    
+    @objc(loginWithUsernameOrEmail:password:realm:audience:scope:parameters:callback:)
+    public func login(withUsernameOrEmail username: String, password: String, realm: String, audience: String, scope: String, parameters: [String: Any]?, callback: @escaping (NSError?, Credentials?) -> Void) {
+        self.authentication
+            .login(usernameOrEmail: username, password: password, realm: realm, audience: audience, scope: scope, parameters: parameters ?? [:])
```

Comment:
Please put the default empty dictionary as a default parameter value, like `parameters: [String: Any]? = [:]`.

Reply:
Moved the default into the parameters list. I was following the code style of the existing loginWithUsernameOrEmail exposed function which used the default in the swift call rather than the exposed function parameters list - happy either way! - I haven't touched the other call so this may need updating too.